src/test/java/bio/terra/service/profile/ProfileAPIControllerTest.java

```diff
@@ -158,6 +158,9 @@ void testDeleteProfile(
     UUID deleteId = UUID.fromString("aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee");
     String jobId = "jobId";
     when(profileService.deleteProfile(deleteId, deleteCloudResources, user)).thenReturn(jobId);
+    if (deleteCloudResources) {
+      when(applicationConfiguration.getResourceId()).thenReturn(UUID.randomUUID().toString());
```

Comment:
In practice I believe the value of `getResourceId()` is always `broad-jade-dev` as set by `application.properties`: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/resources/application.properties#L54

Although using `UUID.randomUUID()` is fine too when testing.

Reply:
I was looking at this again and found a bug! When checking if the profile exists, I was passing in the resource id instead of the profile id. I fixed that here: https://github.com/DataBiosphere/jade-data-repo/pull/1545/commits/363e74bcf65bc618adfc596111815e57deecedb4